nixos/modules/installer/sd-card/sd-image.nix

```diff
@@ -33,6 +32,15 @@ in
   ];
 
   options.sdImage = {
+    makeFsExtraArgs = mkOption {
+      default = {
+        compressImage = true;
+      };
```

Comment:
Move `volumeLabel` here

Reply:
I'd rather say: remove this entirely and use the existing `sdImage.compressImage` that is still present and still won't do anything now.

Reply:
I do believe `sdImage.compressImage` does __one__ thing ATM, it compresses the final image. However the intermediate ext4 partition is always compressed, that's the issue described in https://github.com/NixOS/nixpkgs/issues/151359. I'm fine with whatever solution allows to not call `zstd` at all.